recommender-system/README.md

```diff
@@ -1 +1,135 @@
-# Recommender system
\ No newline at end of file
+# Recommender system
+
+## AWS Setup:
+We want to use Python and the NumPy module in an AWS Lambda function. In this objective, the only solution found was to create a dockerized environnement.
+
+### Container setup with Docker for Python
+1. On your local machine, create a project directory for your new function.
+2. In your project directory, add a file named app.py containing your function code, then add your function handler code to the app.py function. Example 
+```
+import numpy
+
+def handler(event, context):
+    return np.zeros((10,10))
+    
+```
+
+3. In your project directory, add a file named requirements.txt. List each required library as a separate line in this file.
+4. Create a Dockerfile like so:  
+Install any dependencies under the ${LAMBDA_TASK_ROOT} directory alongside the function handler to ensure that the Lambda runtime can locate them when the function is invoked.
```

Comment:
~~any~~, ~~alongside~~
No *Lambda runtime* involves here (well, at least, I don't know what it is in this environment), just import *packages* to use them
${LAMBDA_TASK_ROOT} > `${LAMBDA_TASK_ROOT}`